api/api.pb.txt

```diff
@@ -290,21 +282,6 @@ file {
       }
     }
   }
-  message_type {
```

Comment:
Wondering why these changes are here; this could be if you're using a different version of protobuf (see https://github.com/docker/swarmkit/pull/2503), could you check what version you're using to regenerate?

Reply:
Thank you. Fixed. Seems the build uses protoc 3.6.1 now (on both containerized builds and CircleCI). 

Perhaps the `BUILDING.md` file needs an update, as it refers to `3.x or higher` versions of protoc:
 
https://github.com/docker/swarmkit/blob/master/BUILDING.md#regenerating-protobuf-bindings

I think a link to the option of building in a container, showing up at the top of the `BUILDING.md`, could be useful as well (I can submit a PR for that if it sounds like a good idea)

Reply:
Ah, yes, that would be a good thing to do. I also think that @dperny was working on a containerised version a while back (so that the version could be "pinned" in that container)